app/namespaces/meta_analysis/meta_analysis_controller.py

```diff
@@ -64,6 +64,9 @@ def post(self):
         columns = ['country', 'denominator_value', 'serum_pos_prevalence']
         columns.append(agg_var)
         records = get_filtered_records(filters=filters, columns=columns, start_date=start_date, end_date=end_date)
+        if not records:
+            logging.error('No records with specified filters found.')
+            return dict(records)
```

Comment:
nit: I would just return dict(), I know that they're functionally the same but it's clearer that we're returning an empty dict that way.